mozci/data/sources/taskcluster/__init__.py

```diff
@@ -64,7 +64,7 @@ def run_push_tasks(self, branch, rev):
                 run = result["status"]["runs"][-1]
 
                 # Normalize the state to match treeherder's values.
-                if task["state"] in ("failed", "exception"):
+                if task["state"] == "failed":
```

Comment:
`exception` could also mean a task timed out. Are we 100% sure we won't have errorsummary.log files in those cases? I guess so, just want to double confirm.

Reply:
Yeah, timed out tasks don't have an errorsummary.log. Though this timed out task has stated `failed` rather than `exception`. Which makes it tricky as sometimes `failed` tasks will have errorsummaries and sometimes they won't. I guess the guard from the previous commit is necessary for this reason too then...

Reply:
If you were reading timeout in the taskcluster docs, I think they mean a different timeout when the job fails to start before the claim expires.

Reply:
Tasks can fail with "exception" also when the deadline is exceeded (I've seen it with bugbug data gathering tasks).

Reply:
Interesting.. I wonder what the difference is between those tasks and the one I linked. Either way a timeout should mean no errorsummary as that gets copied to the artifact dir *after* the suite finishes running.